src/app/admin/products/[barcode]/AttachedBoxesList.tsx

```diff
@@ -0,0 +1,52 @@
+"use const";
+
+import Barcode from "@/components/Barcode";
+import { Button } from "@/components/ui/button";
+import { Box } from "@/server/requests/boxRequests";
+import Link from "next/link";
+
+type OwnProps = {
```

Comment:
nit: "AttachedBoxesListProps" is a handful but makes it clear when peeking in editor

Reply:
Keeping as is